content/applications/sales/point_of_sale/overview/getting_started.rst

```diff
@@ -23,44 +24,56 @@ Configure payment methods
+:guilabel:`Cash` to use this payment method for cash payments, or :guilabel:`Bank` to use it for
+card payments.
+
+Check :guilabel:`Identify Customer` to allow this payment method *exclusively* for identified
+customers. For any unidentified customers to be able to pay, leave the :guilabel:`Identify Customer`
+field unchecked.
+
+.. comment:
+   when created, add a link to the customeraccount page under advanced_pricing_features
 
 .. image:: getting_started/payment-method.png
    :align: center
    :alt: Creating a new payment method for a POS.
 
 .. note::
-   Credentials are mandatory to use a payment terminal. To learn how to configure the different
-   terminals, check out the following documentation pages:.
+   - Selecting a **bank** journal automatically adds the :guilabel:`Use a Payment Terminal` field in
+     which you can add your terminal's information.
+   - Credentials are mandatory to use a payment terminal. To learn how to configure the different
+     terminals, check out the following documentation pages:
 
-   - :doc:`Adyen configuration <../payment/adyen>`
-   - :doc:`Vantiv configuration <../payment/vantiv>`
-   - :doc:`Ingenico configuration <../payment/ingenico>`
-   - :doc:`Six configuration <../payment/six>`
+     - :doc:`Adyen configuration <../payment/adyen>`
+     - :doc:`Vantiv configuration <../payment/vantiv>`
+     - :doc:`Ingenico configuration <../payment/ingenico>`
+     - :doc:`Six configuration <../payment/six>`
 
-Now, you can select the payment method in your POS settings. To do so, go to
+Once the payment method is created, you can select it in your POS settings. To do so, go to
 :menuselection:`Point of Sale --> Configuration --> Point of Sale` and select a POS for which you
 wish to make the payment method available. Click :guilabel:`Edit` and add the payment method under
 the :guilabel:`Payments` section.
 
-
 POS sessions
 ============
 
+.. _session:
+
 Start a session
 ---------------
 
-From the **POS dashboard**, click :guilabel:`New Session` to start a POS session, or
-:guilabel:`Resume` if the session was already open.
+From the **POS dashboard**, click :guilabel:`New Session` and :guilabel:`Open Session` at the
+:guilabel:`Opening Cash Control` screen to start a POS session, or click :guilabel:`Continue
+Selling` if the session was already opened.
 
 .. note::
    Multiple users can be connected to the same session at the same time. However, the session can
-   only be open once on the same browser.
+   only be opened once on the same browser.
 
-Click on products to add them to the cart. To change the **quantity**, click :guilabel:`Qty` and
-enter the number of products using the keypad. To add a **discount** or modify the product
-**price**, click respectively :guilabel:`Disc` or :guilabel:`Price` and enter the amounts.
+Click products to add them to the cart. To change the **quantity**, click :guilabel:`Qty` and enter
```

Comment:
```suggestion
Click products to add them to the cart. To change the **quantity**, click on the green button :guilabel:`Qty` and enter
```

Reply:
for consistency, I avoid using "click on the"